js/custom-menu.js

```diff
@@ -6,11 +6,32 @@
 (function ($, Drupal) {
   Drupal.behaviors.cdMenu = {
     attach: function (context, settings) {
-      $('.cd-dropdown .expanded > a').on('click', function (e) {
-        $(this).parent().toggleClass('open');
-        e.stopPropagation();
-        e.preventDefault();
-      });
+
+      // Media query event handler.
```

Comment:
Can you describe the purpose of the handler in the comment? Doesn't need to be extensive but it's the kind of explainer that will probably be useful when we revisit in the future.

Reply:
For sure, added a few lines.